test/sib-test/TestExportGenerators.mss

```diff
@@ -35,21 +35,21 @@ function TestGenerateMEIHeader (assert, plugin) {
 
     m = sibmei.GenerateMEIHeader();
 
-    mei = libmei.Mei();
-    libmei.setDocumentRoot(mei);
-    libmei.AddChild(mei, m);
+    mei = CreateElement('mei');
+    SetDocumentRoot(mei);
+    AddChild(mei, m);
 
-    d = libmei.getDocument();
+    d = GetDocument();
     filePath = Self._property:tempDir & 'header.mei';
-    e = libmei.meiDocumentToFile(d, filePath);
+    e = MeiDocumentToFile(d, filePath);
     assert.OK(e, 'The file ' & filePath & ' was successfully generated');
 
-    libmei.destroy();
+    InitXml();
```

Comment:
Could you say why you renamed this? It seems like `InitXml` should be called before doing something, but this is always called after, which was the original meaning. 

Maybe you don't like `destroy`, but maybe something like `Clear()`?

Reply:
How about ResetXml?